packages/xstate-store/src/types.ts

```diff
@@ -72,7 +74,8 @@ export interface Store<
   TEmitted extends EventObject
 > extends Subscribable<StoreSnapshot<TContext>>,
     InteropObservable<StoreSnapshot<TContext>>,
-    Readable<StoreSnapshot<TContext>> {
+    Readable<StoreSnapshot<TContext>>,
+    BaseAtom<StoreSnapshot<TContext>> {
```

Comment:
BaseAtom is already a Redable so this looks redundant to me:
```suggestion
    BaseAtom<StoreSnapshot<TContext>> {
```